### DISCUSSION TITLE
I Wan to include jupyter notebook into my site, how can i do it, is it possible tell me

### DISCUSSION THREAD

**Discussion by sumendar** [quarto-dev/quarto-cli#6622] [Q&A]
### Description

I Wan to include Jupyter Notebook into my site, how can i do it, is it possible tell me, thank you
**like in this course:** 
https://www.deeplearning.ai/short-courses/chatgpt-prompt-engineering-for-developers/

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/authoring/notebook-embed.html>.
